example/subelements/test_element_functions.cxx

```diff
@@ -0,0 +1,95 @@
+/*

```

Comment:
I changed the name to "t8_test_subelement_functions.cxx". At the moment I have no idea for a better name (especially without "test" in it, since it is a test of the subelement functions..). 

What would I have to add to make this file a real test that checks, if everything works? At the moment, this file creates an element and splits this element into different subelements, depending on a user given subelement type. Printing the coordinates of the subelement vertices and giving some more information afterwards, it is possible to check whether all steps up to this point are working properly.

Reply:
Sidenote: Indent is super messy in this file for some reason

Reply:
To make this file a 'real' test:

- move it to the test/ subfolder and add it to the Makefile.am there
- Use SC_CHECK_ABORT(F) for all the checks in your test (see also how the other files in test/ do it)

Reply:
I merged your changes, the file is now indented. 
I also added a better comment to the file which explains what happens in this file.  
For now, I will leave this file where it is and add it to the test folder in the future (with additional checks).

Reply:
sounds good to me